gap/projective.gi

```diff
@@ -398,10 +398,10 @@ AddMethod(
 AddMethod(
     FindHomDbProjective,
     rec(
-        method := FindHomMethodsProjective.SporadicsByOrders,
+        method := FindHomMethodsProjective.NameSporadic,
         rank := 820,
-        stamp := "SporadicsByOrders",
-        comment := "generate a few random elements and compute the proj. orders",
+        stamp := "NameSporadic",
+        comment := "generate maximal orders",
```

Comment:
Perhaps we should leave the old `AddMethod` in, albeit commented out, to make switching between the two implementations easy (while we are debugging things).

Reply:
Ok. I'm also going to give `NameSporadic` a rank such that it is called more or less directly before `ThreeElementOrders`. `SporadicsByOrders` only generated 100 random elements or so, but `NameSporadic` generates 500 random elements, which sounds like it might be expensive.